zilsd.adoc

```diff
@@ -35,12 +35,17 @@ The Zilsd extension adds the following RV32-only instructions:
 
 [NOTE]
 ====
-As the access size is 64-bit, accesses are only considered naturally aligned for effective addresses that are a multiple of 8. In this case, these instruction are guaranteed to not raise an address-misaligned exception.
+As the access size is 64-bit, accesses are only considered naturally aligned for effective addresses that are a multiple of 8.
```

Comment:
Should also add:
For the purpose of raising misaligned-address or access exceptions, the address is considered to be naturally aligned if it is a multiple of 8.